pycrdt_websocket/yroom.py

```diff
@@ -245,8 +271,12 @@ async def serve(self, websocket: Websocket):
                                 client.path,
                             )
                             tg.start_soon(client.send, message)
-            except Exception as e:
-                self.log.debug("Error serving endpoint: %s", websocket.path, exc_info=e)
+                # remove this client
+                self.clients = [c for c in self.clients if c != websocket]
+        except Exception as exception:
+            self._handle_exception(exception)
 
-            # remove this client
-            self.clients = [c for c in self.clients if c != websocket]
+    def exception_logger(exception: Exception, log: Logger) -> bool:
+        """An exception handler that logs the exception and discards it."""
+        log.error("WebsocketServer exception", exc_info=exception)
+        return True  # the exception was handled
```

Comment:
```suggestion
```
I don't think we need this function, since it's already available in #31.

Reply:
This is also causing the tests to fail, because it needs to be un-indented if we keep it. It's not a method of the class, rather a stand alone function.

Reply:
addressed